src/tools/miri/src/shims/windows/foreign_items.rs

```diff
@@ -792,8 +792,9 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
                     );
                 }
                 // This function looks and behaves excatly like miri_start_unwind.
+                // TODO: this is an unwind: true, would it still be correct after it is changed to Conv::C?
                 let [payload] =
-                    this.check_shim(abi, ExternAbi::C { unwind: true }, link_name, args)?;
```

Comment:
This is the only ``unwind: true`` that I saw. Would it still be correct after we change it to ``Conv::C``?

Reply:
Yes.